yarn.lock

```diff
@@ -43,6 +43,26 @@
     semver "^5.4.1"
     source-map "^0.5.0"
 
+"@babel/core@^7.1.2":
```

Comment:
Should not be here.

Reply:
Seems related to "multiple" version of `@ovh-ux/component-rollup-configuration`.

I will fix this in a separate Pull request.

Thanks for pointing out this.

Reply:
Fixed by #1469